modules/ocf_printhost/files/enforcer

```diff
@@ -147,18 +149,26 @@ def send_printer_mail(message, job, quo):
     send_mail_user(job.user, message.subject, body)
 
 
-def prehook(c, job):
+def prehook(c, r, job):
     quo = quota.get_quota(c, job.user)
     if job.pages > quo.daily:
         send_printer_mail(INSUFFICIENT_QUOTA_MESSAGE, job, quo)
+        msg = job.user + '\nYour print job failed due to insufficient pages.  Your job was ' \
+            + job.pages + 'pages and you have ' + quo.daily + ' pages remaining today.'
```

Comment:
I'd use format strings for this instead, I think that would make this quite a bit cleaner.